packages/core/src/vars.css

```diff
@@ -29,10 +29,10 @@
 	--ts-margin: 0 var(--ts-unit) var(--ts-unit) 0;
 }
 
-html {
-	width: 100%;
-	height: 100%;
-	margin: 0;
-	padding: 0;
-	overflow: hidden;
+ts-root {
+	visibility: hidden;
+}
+
+ts-root[ready] {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the unknown "ts-root" type selector. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aunknown-type-selector)